src/ui/display.c

```diff
@@ -318,15 +318,15 @@ bool prepare_display_output() {
 
     // token_index == 0 means HTR, else use token_index-1 as index on the tokens array
     if (token_index == 0) {
-        strcpy(symbol, "HTR");
+        strlcpy(symbol, "HTR", 4);
```

Comment:
`strcpy` is inherently unsafe due to unbounded buffer, meaning that if the string is not NULL terminated it would read the memory until an error occurs.

`strlcpy` works the same way but we define a limit to how many characters can be read.
This method was chosen due to how it behaves if the limit is reached, it simply puts a `0x00` byte at the end so the buffer is a valid string even if an error happens.